spec/integration/support/inspect_unmanaged_files_examples.rb

```diff
@@ -63,6 +63,7 @@ def parse_md5sums(output)
 
       expected = <<EOF
 Inspecting unmanaged-files...
+Note: Using traditional inspection because there is no helper binary for architecture 'x0_0' available.
```

Comment:
`architecture 'x0_0'` is kind of irritating. Is there a way to improve `normalize_inspect_output` to not mangle the architecture string?